src/entity-schema/EntitySchemaListenerOptions.ts

```diff
@@ -0,0 +1,13 @@
+import {EventListenerType} from "../metadata/types/EventListenerTypes.js";
```

Comment:
can you please remove `.js` extension from all imports in all files?

Reply:
Ah, sure!  A bit of muscle memory from other projects. :)

Might be good to bring in `eslint-plugin-import` & enforce via linting as well.

Reply:
The extensions should be removed.